src/Microsoft.DncEng.SecretManager/SecretTypes/PgpKeyPair.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.DncEng.CommandLineLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("pgp-key-pair")]
+    internal class PgpKeyPair : SecretType<PgpKeyPair.Parameters>
+    {
+        private readonly IConsole _console;
+
+        public class Parameters { }
+    
+        public PgpKeyPair(IConsole console)
+        {
+            _console = console;
+        }
+
+        protected override async Task<SecretData> RotateValue(Parameters parameters, RotationContext context, CancellationToken cancellationToken)
+        {
+            // Adding this to suppress the warning about not having the 'await' operator in an async method
+            await context.GetSecretValue(new SecretReference(context.SecretName));
```

Comment:
Please remove this and just suppress the warning with pragma.
Or even better just return Task.FromResult()